src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -74,85 +144,1336 @@ static DamageTable D_80BBF858 = {
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x66A;
+        case 0x66A:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_TIRED;
+            return 0x66B;
+
+        case 0x677:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x678;
+        case 0x678:
+            return 0x670;
+        case 0x66E:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x66F;
+        case 0x66F:
+            return 0x670;
+        case 0x679:
+            return 0x670;
+        case 0x67A:
+            return 0x670;
+
+        case 0x67E:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x67F;
+        case 0x67F:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            return 0x680;
+        case 0x67B:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x67C;
+        case 0x67C:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            return 0x67D;
+        case 0x681:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            return 0x682;
+        case 0x683:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            return 0x684;
+
+        case 0x670:
+            if (this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_YESBUY) {
+                if (AMMO(ITEM_POWDER_KEG) != 0) {
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    play_sound(NA_SE_SY_ERROR);
+                    return 0x673;
+                }
+                this->powderKegPrice = play->msgCtx.unk1206C;
+                if (gSaveContext.save.playerData.rupees < this->powderKegPrice) {
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_TIRED;
+                    play_sound(NA_SE_SY_ERROR);
+                    return 0x674;
+                }
+                if ((gSaveContext.save.day == 3) && (gSaveContext.save.isNight)) {
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    func_8019F208();
+                    return 0x676;
+                }
+                this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                func_8019F208();
+                return 0x675;
+            }
+            if ((gSaveContext.save.day == 3) && (gSaveContext.save.isNight)) {
+                this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                return 0x672;
+            }
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            return 0x671;
+    }
+    return 0x0;
+}
+
+/**
+ * Links to actor of type EN_JG (Goron Elder), returns whether operation successful.
+ */
+s32 EnSGoro_SetPartnerActor_GoronElder(EnSGoro* this, PlayState* play) {
```

Comment:
We try to only separate with extra underscores if it is a type of subsystem (like how there are different types of gorons here). Personally I think something like `EnSGoro_FindGoronElder` and `EnSGoro_FindGoronChild` would work.